keras_cv/bounding_box/converters.py

```diff
@@ -236,6 +236,10 @@ def convert_format(boxes, source, target, images=None, dtype="float32"):
     if source == target:
         return boxes
 
+    if "rel" in source and "rel" in target:
+        source = source[4:]
```

Comment:
can we make this check "startsWith" instead of "in"?  

also, can we use python string.replace()?   It supports a `count` param:

source = source.replace("rel_", "", 1)